model/clusters_mgmt/v1/event_resource.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific trackable event.
+resource Event {
```

Comment:
What is the purpose of this resource? It has no methods, and the locator seems wrong, it corresponds to an URL path with a loop like this:

```
/api/clusters_mgmt/v1/events/{key}/events/{key}/events/...
```

I'd say this isn't needed.

Reply:
Removed